Jenkinsfile

```diff
@@ -92,6 +107,10 @@ pipeline {
     }
 
     post {
+        always {
+            echo 'Cleaning up fabric network'
+            sh './scripts/start_fabric.sh -c'
```

Comment:
I would bring down Fabric with `./scripts/start_fabric.sh -d` before running `-c` to clean it up. 
It seems cleaner to do that.
Is there a reason not to run `./scripts/start_fabric.sh -d`  ?

Reply:
-c will bring down the fabric container and also cleanup of work_dir where fabric certificates stored.